core/src/dunit/scala/io/snappydata/cluster/SplitClusterDUnitTestBase.scala

```diff
@@ -393,6 +395,18 @@ trait SplitClusterDUnitTestObject extends Logging {
     }
     value
   }
+  def validateNoActiveSnapshotTX(): Unit = {
```

Comment:
@suranjan and @SachinJanani shall we add this to tear down for all dunit test and junit test, such that it gets invoked in all the vms.